projects/packages/my-jetpack/_inc/admin.jsx

```diff
@@ -48,7 +48,7 @@ const MyJetpack = () => (
 			<Routes>
 				<Route path="/" element={ <MyJetpackScreen /> } />
 				<Route path="/connection" element={ <ConnectionScreen /> } />
-				<Route path="/add-anti-spam" element={ <AntiSpamInterstitial /> } />
+				<Route path="/add-akismet" element={ <AntiSpamInterstitial /> } />
```

Comment:
Changed this to match the convention of `add-` followed by the product key, which is followed by all other products (see projects/plugins/jetpack/_inc/client/product-descriptions/constants.js). The product key is 'akismet' not 'anti-spam'.

Reply:
Would it be better if we keep the old route and add a new one, so that links shared anywhere before should still work.

Reply:
That sounds like a good solution - I was slightly concerned about removing the old route although it didn't seem like it was use anywhere. Best to be safe!

Reply:
Fixed in [47e54dd](https://github.com/Automattic/jetpack/pull/29598/commits/47e54dd19446d427a595152b22fac9ae3795de4e).